<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameWidget</class>
 <widget class="QMainWindow" name="GameWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1350</width>
    <height>1043</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>1350</width>
     <height>1000</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1350</width>
     <height>1000</height>
    </size>
   </property>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1141</width>
      <height>121</height>
     </rect>
    </property>
    <widget class="QProgressBar" name="progressbar">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>50</y>
       <width>700</width>
       <height>30</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>700</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>700</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximum">
      <number>1500</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="textVisible">
      <bool>false</bool>
     </property>
     <property name="textDirection">
      <enum>QProgressBar::TopToBottom</enum>
     </property>
    </widget>
    <widget class="QLabel" name="scoreLabel">
     <property name="geometry">
      <rect>
       <x>980</x>
       <y>50</y>
       <width>54</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="scoreLabel_3">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>50</y>
       <width>54</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="scoreLabel_2">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>50</y>
       <width>54</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="timerLabel">
    <property name="geometry">
     <rect>
      <x>1223</x>
      <y>50</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1350</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>游戏</string>
    </property>
    <widget class="QMenu" name="menu_3">
     <property name="title">
      <string>读档</string>
     </property>
     <addaction name="actionload1"/>
    </widget>
    <widget class="QMenu" name="menu_2">
     <property name="title">
      <string>存档</string>
     </property>
     <addaction name="separator"/>
     <addaction name="actionsave1"/>
    </widget>
    <addaction name="separator"/>
    <addaction name="menu_2"/>
    <addaction name="menu_3"/>
    <addaction name="separator"/>
    <addaction name="actionpause"/>
    <addaction name="actiongoon"/>
    <addaction name="separator"/>
    <addaction name="actionreturn"/>
    <addaction name="actionquit"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionquit">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="actionpause">
   <property name="text">
    <string>暂停</string>
   </property>
  </action>
  <action name="actionsave1">
   <property name="text">
    <string>save1</string>
   </property>
  </action>
  <action name="actionsave2">
   <property name="text">
    <string>save2</string>
   </property>
  </action>
  <action name="actionsave3">
   <property name="text">
    <string>save3</string>
   </property>
  </action>
  <action name="actionsave4">
   <property name="text">
    <string>save4</string>
   </property>
  </action>
  <action name="actionload1">
   <property name="text">
    <string>load1</string>
   </property>
  </action>
  <action name="actionload2">
   <property name="text">
    <string>load2</string>
   </property>
  </action>
  <action name="actionload3">
   <property name="text">
    <string>load3</string>
   </property>
  </action>
  <action name="actionload4">
   <property name="text">
    <string>load4</string>
   </property>
  </action>
  <action name="actiongoon">
   <property name="text">
    <string>继续</string>
   </property>
  </action>
  <action name="actionreturn">
   <property name="text">
    <string>返回</string>
   </property>
  </action>
  <action name="actionreset">
   <property name="text">
    <string>重启</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
